driver/src/contracts/snapp_contract.rs

```diff
@@ -311,249 +316,3 @@ where
         Ok(())
     }
 }
-
-#[cfg(test)]
-pub mod tests {
```

Comment:
Really nice to see that all this is no longer necessary. You are becoming quite the "rustacean"!